index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<head>
+    <meta charset="utf-8">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body onload="getUsers()">
+    <div class="container">
+        <table id="table" class="table">
+        </table>
```

Comment:
there is no <tbody> tag, so DOM api will add it. This leads to errors in markup and styles, as each row styled as separate table. Fix it and pay attention on how will change your table visually